### PR TITLE
open-cluster-management/backlog#5396 New adds

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: manage_cluster/custom_resource.adoc [hunk 1]
```diff
@@ -1,64 +1,113 @@
...
++
+Replace _clusterset1_ with the name of your `ManagedClusterSet`. 
 
-* Create a ManagedClusterSet resource that contain a list of labels. Your resource might resemble the following information:
+[#adding-clusters-to-a-managedclusterset]
+== Adding clusters to a ManagedClusterSet
 
+After your `ManagedClusterSet` is created, you must add one or more managed clusters. Complete the following steps to add managed clusters:
+
+**Note:** You must have cluster administrator access to all of the managed clusters that you add to the `ManagedClusterSet`, and to the namespace where the `ManagedClusterSet` is hosted.
```
Comment (elgnay) [inline manage_cluster/custom_resource.adoc]: ManagedClusterSet is a cluster scope resource. No namespace is needed to host it.

File: manage_cluster/custom_resource.adoc [hunk 2]
```diff
@@ -1,64 +1,113 @@
...
++
+----
+apiVersion: cluster.open-cluster-management.io/v1
+kind: ManagedCluster
+metadata:
+  name: cluster1
+  labels:
+spec:
+  hubAcceptsClient: true 
+----
```
Comment (elgnay) [inline manage_cluster/custom_resource.adoc:62]: - The empty labels can be removed;
```
apiVersion: cluster.open-cluster-management.io/v1
kind: ManagedCluster
metadata:
  name: cluster1
spec:
  hubAcceptsClient: true
```
- It should be noted that the above sample yaml file does not show the whole definition of a ManagedCluster.

File: manage_cluster/custom_resource.adoc (hunk 1 above)
Comment (cadawson) [reply manage_cluster/custom_resource.adoc]: Removed namespace

File: manage_cluster/custom_resource.adoc (hunk 2 above)
Comment (cadawson) [reply manage_cluster/custom_resource.adoc:62]: Removed empty label and referred to the section of the code.

**Comment by elgnay** [comment]
The doc looks good to me.

**Comment by cadawson** [comment]
Thank you @elgnay !

**Pull request merged**